sui/its-example.js

```diff
@@ -0,0 +1,499 @@
+const { Command, Option } = require('commander');
+const { ITSMessageType, SUI_PACKAGE_ID, CLOCK_PACKAGE_ID, TxBuilder, copyMovePackage } = require('@axelar-network/axelar-cgp-sui');
+const { loadConfig, saveConfig, printInfo } = require('../common/utils');
+const {
+    parseExecuteDataFromTransaction,
+    addBaseOptions,
+    addOptionsToCommands,
+    getUnitAmount,
+    getWallet,
+    findPublishedObject,
+    printWalletInfo,
+    getObjectIdsByObjectTypes,
+    broadcastFromTxBuilder,
+    moveDir,
+    getTransactionList,
+    checkTrustedAddresses,
+} = require('./utils');
+const { ethers } = require('hardhat');
+const {
+    utils: { defaultAbiCoder, keccak256, toUtf8Bytes, hexlify, randomBytes },
+} = ethers;
+
+async function sendToken(keypair, client, contracts, args, options) {
+    const [symbol, destinationChain, destinationAddress, feeAmount, amount] = args;
+
+    const { Example, GasService, AxelarGateway, ITS } = contracts;
+    const ItsToken = contracts[symbol.toUpperCase()];
+
+    if (!ItsToken) {
+        throw new Error(`Token ${symbol} not found. Deploy it first with 'node sui/its-example.js deploy-token' command`);
+    }
+
+    checkTrustedAddresses(ITS.trustedAddresses, destinationChain, destinationAddress);
+
+    const decimals = ItsToken.decimals;
+
+    const unitAmount = getUnitAmount(amount, decimals);
+    const unitFeeAmount = getUnitAmount(feeAmount);
+    const walletAddress = keypair.toSuiAddress();
+
+    const objectIds = {
+        singleton: Example.objects.ItsSingleton,
+        its: ITS.objects.ITS,
+        gateway: AxelarGateway.objects.Gateway,
+        gasService: GasService.objects.GasService,
+    };
+
+    const txBuilder = new TxBuilder(client);
+
+    const tx = txBuilder.tx;
+
+    const gas = tx.splitCoins(tx.gas, [unitFeeAmount]);
+
+    const TokenId = await txBuilder.moveCall({
+        target: `${ITS.address}::token_id::from_u256`,
+        arguments: [ItsToken.objects.TokenId],
+    });
+
+    const Coin = await txBuilder.moveCall({
+        target: `${SUI_PACKAGE_ID}::coin::mint`,
+        arguments: [ItsToken.objects.TreasuryCap, unitAmount],
+        typeArguments: [ItsToken.typeArgument],
+    });
+
+    await txBuilder.moveCall({
+        target: `${Example.address}::its::send_interchain_transfer_call`,
+        arguments: [
+            objectIds.singleton,
+            objectIds.its,
+            objectIds.gateway,
+            objectIds.gasService,
+            TokenId,
+            Coin,
+            destinationChain,
+            destinationAddress,
+            '0x', // its token metadata
+            walletAddress,
+            gas,
+            '0x', // gas params
+            CLOCK_PACKAGE_ID,
+        ],
+        typeArguments: [ItsToken.typeArgument],
+    });
+
+    await broadcastFromTxBuilder(txBuilder, keypair, `${amount} ${symbol} Token Sent`);
+}
+
+async function receiveToken(keypair, client, contracts, args, options) {
```

Comment:
Maybe add an function that receives some coin without it being routed to a module (example in this case)